core/cloudflow-akka/src/main/scala/cloudflow/akkastream/AkkaStreamletContextImpl.scala

```diff
@@ -45,7 +45,8 @@ final class AkkaStreamletContextImpl(
     private[cloudflow] override val streamletDefinition: StreamletDefinition,
     sys: ActorSystem
 ) extends AkkaStreamletContext {
-  implicit val system: ActorSystem = sys
+  implicit val system: ActorSystem  = sys
+  implicit val ec: ExecutionContext = sys.dispatcher
```

Comment:
Should this not be an implicit on the method that uses it?

Reply:
actually this is already provided on `AkkastreamletLogic` trait.

Reply:
Is the new way I have implemented look better?